module/functions/azure/azcli/Invoke-AzCliRestCommand.ps1

```diff
@@ -35,22 +38,29 @@ The JSON output from the underlying azure-cli command, in hashtable format.
 
 function Invoke-AzCliRestCommand
 {
-    [CmdletBinding()]
+    [CmdletBinding(DefaultParameterSetName = 'Body as hashtable')]
     param (
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true, ParameterSetName = 'Body as hashtable')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'Body as file')]
         [string] $Uri,
         
-        [Parameter()]
+        [Parameter(ParameterSetName = 'Body as hashtable')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'Body as file')]
```

Comment:
This one shouldn't be mandatory for the 'body as file' set, as we still offer a default value.